src/peer.js

```diff
@@ -147,7 +147,7 @@ class Peer extends EventEmitter {
 
   ping (cb) {
     var start = Date.now()
-    var nonce = crypto.pseudoRandomBytes(8)
+    var nonce = crypto.randomBytes(8)
```

Comment:
I'm not sure I agree with this change, calling `randomBytes` can block for a long time if there isn't sufficient entropy. We totally don't need cryptographic randomness here either.

Are you sure webpack has issues? That would be strange since it would break a lot of code. If there really are compatibility issues I would prefer moving to some module that provides the randomness.